client/src/App.tsx

```diff
@@ -83,6 +88,136 @@ const App = () => {
         document.title = (startingIn > 0)
             ? `${name} starting in ${startingIn} minute${startingIn !== 1 ? 's' : ''}.`
             : `${name} ending in ${endingIn} minute${endingIn !== 1 ? 's' : ''}, started ${-startingIn} minute${startingIn !== -1 ? 's' : ''} ago.`
+
+        /** ====== begin favicon ====== */
+
+        const favicon = document.createElement('link');
+        favicon.setAttribute('rel', 'icon')
+        document.head.appendChild(favicon);
```

Comment:
Hmm, do you think this would conflict with an existing favicon in `<head>` on some browsers?

Edit: LOL Markdown removed `<head>` when it wasn't in backticks